doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -35,6 +35,13 @@ IDE, and tool support
 
 |no_changes_yet_note|
 
+Board support
+=============
+
+* Added support for the Thingy:91 X board.
```

Comment:
@bjornspock, @gregersrygg, @maxd-nordic - Please confirm if it is okay to mention about the Thingy:91 X board support in the current release notes.